### DISCUSSION TITLE
Image refresh after update the quantity

### DISCUSSION THREAD

**Discussion by Elkaroui** [EvgeniyMukhamedjanov/liquid-ajax-cart#12] [Q&A] [answered]
Hi there,
first I want to thank you for this js library, it's easy to add and work well, I do have a question or an issue that I am facing now when I added product image, it refreshes or blanks each time I update the quantity, how we can fix that?

![image](https://user-images.githubusercontent.com/11858227/156709762-428b2d68-9c65-42af-a2f6-ee233f67367e.png)

thanks

**Comment by EvgeniyMukhamedjanov** [comment, marked as answer]
Hey, thank you for reaching out!
It might happen if your Chrome Dev Tools is opened and you have checked the "Disable Cache" checkbox under the "Network" tab in the Chrome Dev Tools. It occurs because the HTML of the cart section gets replaced with a new HTML and the browser reloads the images instead of taking them from the cache since the "Disable Cache" is checked.
So if you close Chrome Dev Tools the issue should disappear.

If the issue appears even when the Chrome Dev Tool is closed, share a preview link and the code of the cart section. 

Thanks!

**Reply by Elkaroui** [reply]
Thank you @EvgeniyMukhamedjanov, it's working well after closing the chrome inspect element 😊

**Reply by EvgeniyMukhamedjanov** [reply]
Hey @Elkaroui . I want to let you know that I've just published the "Built with Liquid Ajax Cart" section on the [liquid-ajax-cart.js.org](https://liquid-ajax-cart.js.org/) website. Once you finish your project I would be happy to add it there. Let me know if it is possible! GitHub stars are also appreciated :-)

**Reply by Elkaroui** [reply]
@EvgeniyMukhamedjanov I really appreciate it, I had followed your explanation from the website, honestly, it's well explained and with live examples bravo! ⭐

I hope we have a Vue version in the future 😁
